src/app/lmem/contributor.ts

```diff
@@ -11,16 +12,12 @@ interface Picture {
 
 export type ContributorId = number;
 
-export interface PinnedNotice {
+export interface PinnedNotice extends NoticeItem {
   sort?: number;
-  exampleMatchingUrl: string;
-  noticeId: number;
-  noticeUrl: string;
 }
 
 interface Contribution {
-  example: PinnedNotice; // @deprecated use `pinnedNotices` instead
-  starred: PinnedNotice; // @deprecated use `pinnedNotices` instead
+  example: NoticeItem;
```

Comment:
Do we still use this property ? If not, we could remove it since it’s deprecated …

Reply:
`example` it's not actually deprecated, I thought it was but it's not, but will certainly be in the future.

Reply:
Where is it used ?

Reply:
Don't remember at the moment, but I'll dig this and create an issue.
Could we merge this fix and discuss this later?

Reply:
Definitely, no blocker !